scene/main/viewport.cpp

```diff
@@ -491,6 +491,8 @@ void Viewport::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_WM_WINDOW_FOCUS_OUT: {
+			_gui_cancel_tooltip();
+			
```

Comment:
There seems to be trailing whitespace on this line as our formatting checks failed.

Please amend the commit with `git commit --amend` to remove whitespace from this line, and update the PR with `git push --force`.

Reply:
Applied (I hope)

Reply:
I did not do, I made a small error. please wait

Reply:
error resolved it should be okay now, sorry about that